### PR TITLE
testing duckdb-engine rc

### PR DESCRIPTION
**Pull request by edublancas**
## Issue ticket number and link
Closes #x

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] I have added thorough [tests](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#testing) (when necessary).
- [ ] I have added the right documentation in the [docstring](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#documenting-changes-and-new-features) and [changelog](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#changelog) (when needed)

----
:books: Documentation preview :books:: https://jupysql--220.org.readthedocs.build/en/220/

### CODE REVIEW

File: .github/workflows/ci-integration-db.yaml [hunk 1]
```diff
@@ -57,6 +57,7 @@ jobs:
       - name: Install dependencies
         run: |
 
+          pip install git+https://github.com/Mause/duckdb_engine@v0.7.0-rc1
```
Comment (Mause) [inline .github/workflows/ci-integration-db.yaml:60]:
```suggestion
          pip install duckdb_engine==v0.7.0-rc1 --pre
```

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -33,6 +33,7 @@ jobs:
       - name: Install dependencies
         run: |
 
+          pip install git+https://github.com/Mause/duckdb_engine@v0.7.0-rc1
```
Comment (Mause) [inline .github/workflows/ci.yaml:36]:
```suggestion
          pip install duckdb_engine==v0.7.0-rc1 --pre
```

**Review by Mause — CHANGES_REQUESTED** [review]

**Comment by edublancas** [comment]
hey @Mause: I added your suggestion on how to install the release candidate but some tests failed.

however, the error seems unrelated to duckdb-engine

```
 FAILED src/tests/test_plot.py::test_boxplot_stats - duckdb.HTTPException: HTTP Error: HTTP Error: Failed to download extension "sqlite_scanner" at URL "http://extensions.duckdb.org/ab9736bed0/osx_amd64/sqlite_scanner.duckdb_extension.gz"
Extension "sqlite_scanner" is an existing extension.

Are you using a development build? In this case, extensions might not (yet) be uploaded.
FAILED src/tests/test_plot.py::test_boxplot_stats_exception - duckdb.HTTPException: HTTP Error: HTTP Error: Failed to download extension "sqlite_scanner" at URL "http://extensions.duckdb.org/ab9736bed0/osx_amd64/sqlite_scanner.duckdb_extension.gz"
Extension "sqlite_scanner" is an existing extension.

Are you using a development build? In this case, extensions might not (yet) be uploaded.
```


But running the CI from a clean copy from the main branch didn't break: https://github.com/ploomber/jupysql/pull/224

any thoughts?

**Comment by edublancas** [comment]
hi @mause: we found an issue with our fixed for sqlalchemy 2 compatibility (https://github.com/ploomber/jupysql/issues/234); so currently that's blocking us from testing this

**Comment by edublancas** [comment]
hey @Mause, apologies for the long delay. we assigned sqlalchemy 2 compatibility to a contractor, but I didn't finish the job, so we assigned to somebody else in our team, and we just merged it. I see you released duckdb-engine 0.7 already, and our CI is passing with that version, so all good!

**Pull request closed without merging**